simpleflow/swf/mapper/models/base.py

```diff
@@ -19,9 +20,13 @@ class ModelDiff:
     :param  input: triples (tuples) storing in order: compared attribute name,
                    local model attribute value, upstream model attribute value.
     :type   input: *args
+
+    :param  ignore_fields: list of fields to ignore when comparing local and upstream
+    :type   ignore_fields: list
     """
 
-    def __init__(self, *input):
+    def __init__(self, *input, ignore_fields: List[str] = None):
+        self.ignore_fields = ignore_fields or []
         self.container: OrderedDict[str, tuple[str, str]] = self._process_input(input)
```

Comment:
(Minor) Do we still need to support Python < 3.7, or could we %s/OrderedDict/dict/g?

Reply:
You know better, but the pyproject.toml file says "python >= 3.7". If "dict" behaves the same way as OrderedDict in 3.7+ (I didn't know), then we can drop OrderedDict yay. I would do that in a later PR though.

Reply:
https://mail.python.org/pipermail/python-dev/2017-December/151283.html 🙂
And I agree: this is also for a later cleanup PR

Reply:
👍  => #426